ide/static/js/panZoom.js

```diff
@@ -68,14 +63,15 @@ export default function() {
       canvas.y += e.pageY - previousMousePosition.y;
       canvas.updateContainerPosition();
       previousMousePosition = { x: e.pageX, y: e.pageY };
-      //instance.repaintEverything();
     }
   };
 
+  panZoom.onT
+
   panZoom.ondblclick = function(e) {
     e.preventDefault();
-    onZoom((e.ctrlKey || e.metaKey) ? current.zoom * 1.7 * 1.7 : current.zoom / 1.7 / 1.7,
-     e.clientX - panZoom.offsetLeft, e.clientY - panZoom.offsetTop);
+    onZoom((e.ctrlKey || e.metaKey) ? current.zoom * 1.2 * 1.2 : 
+	current.zoom / 1.2 / 1.2, e.clientX - panZoom.offsetLeft, e.clientY - panZoom.offsetTop);
```

Comment:
– If I zoom in with a double click, I'm unable to pinch out to zoom out.
– To confirm, with a mouse, are double-clicking/using '[' or ']' the only zooming hotkeys?